apps/fabric-website/src/components/App/App.tsx

```diff
@@ -165,6 +168,12 @@ export class App extends React.Component<IAppProps, any> {
     });
   };
 
+  private _extractAnchorLink(path) {
+    let split = path.split('#');
+    const cleanedSplit = split.filter(value => !!value);
+    return cleanedSplit[cleanedSplit.length - 1];
```

Comment:
Probably unlikely, but do you need to guard against the case where the array can be empty?

Reply:
Pretty sure `cleanedSplit[-1]` would just return undefined